lib/js-preprocessor/broccoli-generate-components.js

```diff
@@ -60,8 +60,9 @@ GenerateComponents.prototype.appComponentFor = function(fileName) {
   return "export { default } from '" + defaultModule + "';\n";
 };
 
-var componentsToGenerate = function(baseDir, scss) {
+var componentsToGenerate = function(baseDir, scss, modulePrefix) {
   var scssFiles = walkSync(baseDir, { globs: scss });
+  var baseKindFile = path.join(modulePrefix, 'components', 'ui-kind').replace('modules/', '');
```

Comment:
@ianstarz i had to do this to get the import paths to be correct in the addon component files :(